.github/workflows/javascript-typescript.yml

```diff
@@ -66,6 +66,15 @@ jobs:
 
         cd -
 
+    - name: Publish task
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
@thewtex you will need to add an NPM_TOKEN to the repo secrets in order to publish.

Do you want to also publish to github? If not, we can disable the github plugin.

Reply:
:1st_place_medal: :rocket: :man_singer: 

NPM_TOKEN added.

Sure, let's publish to GitHub, too.